engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/MainMenuScreen.java

```diff
@@ -34,7 +34,12 @@
     @Override
     public void initialise() {
         find("version", UILabel.class).setText(TerasologyVersion.getInstance().getHumanVersion());
-        WidgetUtil.trySubscribe(this, "singleplayer", button -> getManager().pushScreen("engine:selectGameScreen"));
+        WidgetUtil.trySubscribe(this, "singleplayer", button -> {
+            UIScreenLayer screen = getManager().pushScreen("engine:selectGameScreen");
```

Comment:
The exact method call is: [NUIManager.pushScreen(String screenUri, Class<T> expectedType);](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/rendering/nui/NUIManager.java#L66). It returns a `SelectGameScreen` you can use directly.